### ISSUE TITLE
Has this library been replaced?

### ISSUE THREAD

**Issue by fruchtose** [kisenka/svg-sprite-loader#318]
**Do you want to request a feature, report a bug or ask a question?**
Question

**What is the current behavior?**
I noticed that there is a new library named [extract-svg-sprite-webpack-plugin](https://github.com/JetBrains/svg-mixer/tree/master/packages/extract-svg-sprite-webpack-plugin). It looks a lot like this library, except it depends on `svg-mixer` instead of `svg-baker`. Is this library currently under development?

**What is the expected behavior?**
Could you clarify the relationship between `extract-svg-sprite-webpack-plugin` and this library?

**Comment by kisenka** [comment]
Hi @fruchtose! Thanks for the question :) Yes, the most complex part of this loader - sprite extracting was moved to extract-svg-sprite-webpack-plugin, which is under development, but already usable. You can try it to extract svg sprites. This loader will move to svg-mixer project in next major release and extract feature will be removed from it. Now I rather busy, but I'am planning to do it at December.

P.S. Maybe someone can advice a handy way to communicate with my users? Github issues is not convenient way to do this. The main goal is to inform my users about such plans.

**Comment by joepagan** [comment]
@kisenka I think just a note at the top of this repo's readme would be best. 
Thanks for your work on this tool.

**Comment by Bosch-Eli-Black** [comment]
Hi @kisenka,

Are you still planning to move this library to use svg-mixer? I'm mostly wondering because this project depends on the svg-baker and svg-baker-runtime packages, and those packages don't have LICENSE files.

Anyway, thanks for your work on this project; just checking :)

**Comment by kisenka** [comment]
@Bosch-Eli-Black yep, I'm still planning to do that, but have no time at this point :(

**Comment by Bosch-Eli-Black** [comment]
@kisenka No worries! :)

I think the main issue for us is that it looks like maybe the LICENSE files disappeared from the svg-baker and svg-baker-runtime libraries.

Would it be possible to just add some LICENSE files to those packages?
